pontoon/settings/base.py

```diff
@@ -728,6 +728,18 @@ def _allowed_hosts():
     'vi-vn', 'wo-sn', 'xh-za', 'yo-ng', 'zh-cn', 'zh-hk', 'zh-tw', 'zu-za',
 ]
 
+# Bleach tags and attributes
+ALLOWED_TAGS = [
+    'a', 'abbr', 'acronym', 'b', 'blockquote', 'br', 'code', 'em', 'i',
+    'li', 'ol', 'p', 'strong', 'ul',
+]
```

Comment:
OK, let's keep this explicit. It should help with figuring out which elements we need to take care of when styling those outputs (I tested all of them and they look OKish).